--- a/modules/initrd.nix
+++ b/modules/initrd.nix
@@ -407,10 +407,6 @@
       mobile.boot.stage-1.environment = {
         LD_LIBRARY_PATH = "${extraUtils}/lib";
         PATH = "${extraUtils}/bin";
       };
-      system.systemBuilderCommands = ''
-        printf ":: Copying Mobile NixOS stage-1...\n"
-        ln -f -v -s "${config.system.build.initialRamdisk}/${config.system.boot.loader.initrdFile}" $out/initrd
-      '';
     };
   }
